context.go

```diff
@@ -123,3 +123,34 @@ func SetHTTPResponseHeader(ctx context.Context, key, value string) error {
 
 	return nil
 }
+
+// AddHTTPResponseHeader adds an HTTP header key-value pair using a context
+// provided by a twirp-generated server, or a child of that context.
+// The server will include the header in its response for that request context.
+//
+// This can be used to respond with custom HTTP headers like "Cache-Control".
+// But note that HTTP headers are a Twirp implementation detail,
+// only visible by middleware, not by the clients or their responses.
+//
+// The header will be ignored (noop) if the context is invalid (i.e. using a new
+// context.Background() instead of passing the context from the handler).
+//
+// If called multiple times with the same key, it appends to any existing values
+// associated with that key.
+//
+// AddHTTPResponseHeader returns an error if the provided header key
+// would overwrite a header that is needed by Twirp, like "Content-Type".
```

Comment:
```
// AddHTTPResponseHeader behaves like SetHTTPResponseHeader,
// but it appends the key-value pair to the header (instead of replacing it).
//
// AddHTTPResponseHeader returns an error if the key is "Content-Type".
```

Reply:
Thanks!
https://github.com/twitchtv/twirp/pull/227/commits/720f86f0b23d8233fc0524fda94d015a578113f9

Reply:
Thanks!
https://github.com/twitchtv/twirp/pull/227/commits/720f86f0b23d8233fc0524fda94d015a578113f9